TimeReport.py

```diff
@@ -76,6 +76,12 @@ def peopleTime(date):
 				enteredTime += hours
 			if enteredTime >= hoursForAcceptance:
 				marked = True
+			# Custom check for interns
+			if args.interns:
+                                internName = first + " " + last
+                                # Checks whether person is an intern and if they have worked yesterday
+                                if internArgs.get(internName) and dayOfWeek in internArgs.get(internName) and enteredTime > 0:
```

Comment:
Can we move this call up and save the result of internArgs.get(InternName) to a variable so we aren't double calling?